files/image_config/process-reboot-cause/process-reboot-cause

```diff
@@ -82,6 +83,8 @@ def find_software_reboot_cause():
         log_info("Reboot cause file {} not found".format(REBOOT_CAUSE_FILE))
 
     if os.path.isfile(FIRST_BOOT_PLATFORM_FILE):
+        if software_reboot_cause == REBOOT_CAUSE_UNKNOWN:
+            software_reboot_cause = REBOOT_CAUSE_SONIC_IMAGE_INSTALL
```

Comment:
because image installation does not trigger the reboot, for example, people can install the image and not trigger reboot. i feel it is a little bit misleading to treat this as a reboot reason. What I feel lacking here is that this first boot file is generated in the new image, not from the previous image. 

can we name as reboot reason "Unknown (First new image boot)"

Reply:
Point taken. Check my latest commit. I also included the version number; might be helpful for debugging issues (e.g., we should only ever see it once).